usr/src/test/zfs-tests/tests/functional/cli_root/zfs_program/cleanup.ksh

```diff
@@ -0,0 +1,30 @@
+#!/bin/ksh -p
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
```

Comment:
nit: but while we're copying... opensolaris.org is long gone. This is a better URL:
https://opensource.org/licenses/CDDL-1.0

Reply:
I guess I can update the license link though